### PR TITLE
Decouple from Templating Component, Stage 1

### PR DESCRIPTION
**Pull request by covex-nn**
I am targeting this branch, because this is a patch.

## Changelog

```markdown
### Changed
- Switch all templates references to Twig namespaced syntax
- Switch from templating service to sonata.templating
```

## To do

- [X] Switch all templates references to Twig namespaced syntax. Update docs, src and tests
- [x] Add `sonata-project/block-bundle` to composer.json
- [x] Switch from `templating` service to `sonata.templating`

## Subject

This PR is a part of a big plan of decoupling sonata-project bundles from Templating Component. See sonata-project/dev-kit#380 for details

### CODE REVIEW

**Comment by dbu** [comment]
the build error does not look related to your pull request. so far this looks good to me, thanks!

**Comment by jordisala1991** [comment]
The build error is the same we are getting on master probably, this is a missing dependency on some bundle...

**Comment by covex-nn** [comment]
Difference between failed and jobs is in installed packages:

| installed package | [failed](https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/jobs/330260948), SONATA_ADMIN=3.* | [passed](https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/jobs/330260949), SONATA_BLOCK=3.* |
| --- | --- | --- |
| [jms/serializer-bundle](https://packagist.org/packages/jms/serializer-bundle) | 1.5.0 | 2.3.1 |
| [symfony/translation](https://packagist.org/packages/symfony/translation) | 4.0.3 | 3.4.3 |
| [symfony-cmf/resource-rest-bundle](https://packagist.org/packages/symfony-cmf/resource-rest-bundle) | 1.0.0 | 1.0.1 |

I did not understand how you did it =)

**Comment by dbu** [comment]
thanks for the fix @covex-nn . @jordisala1991 when you rebase, the build should become green. (but its still WIP anyways, only your first step is checked, so i won't merge yet)

**Comment by covex-nn** [comment]
@dbu there are 15 PRs for `sonata-project/*-bundle` in WIP right now with similar changes =) waiting for BlockBundle release

File: src/Builder/ListBuilder.php [hunk 1]
```diff
@@ -161,19 +161,19 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
             $fieldDescription->setTemplate($this->getTemplate($fieldDescription->getType()));
 
             if (ClassMetadata::MANY_TO_ONE == $fieldDescription->getMappingType()) {
-                $fieldDescription->setTemplate('SonataAdminBundle:CRUD/Association:list_many_to_one.html.twig');
+                $fieldDescription->setTemplate('@SonataAdmin/CRUD/Association/list_many_to_one.html.twig');
             }
 
             if (ClassMetadata::MANY_TO_MANY == $fieldDescription->getMappingType()) {
-                $fieldDescription->setTemplate('SonataAdminBundle:CRUD/Association:list_many_to_many.html.twig');
+                $fieldDescription->setTemplate('@SonataAdminBundle/CRUD/Association/list_many_to_many.html.twig');
```
Comment (jordisala1991) [inline src/Builder/ListBuilder.php]: wrong

Comment (covex-nn) [reply src/Builder/ListBuilder.php]: Fixed. Sorry =)

**Pull request merged**